server.go

```diff
@@ -59,27 +63,40 @@ func (gs *CasbinServerHandler) serve(w http.ResponseWriter, r *http.Request) {
 		glog.Errorf("Filed to load the policies: %v", err)
 		return
 	}
-	if e.HasPermissionForUser(user, operation_name) == true {
-		response := v1.AdmissionReview{
+	/*	if e.HasPermissionForUser(user, operation_name) == true {
+		aresponse := v1.AdmissionReview{
 			Response: &v1.AdmissionResponse{
 				Allowed: true,
 			},
 		}
+		return }
+	*/
```

Comment:
Are you commenting code manually? I recommend using shortcuts from IDEs like Goland or VSCode to comment lines. The manual commenting seems very ad-hoc to me.

![image](https://user-images.githubusercontent.com/3787410/124001729-82dbc800-da07-11eb-893f-e8708012e0f0.png)

Reply:
Yeah I commented it manually. But No problem,I will keep that in mid. I am removing the comments now.